### PR TITLE
feat: build very basic summary key stats table with dummy data

### PR DESCRIPTION
**Pull request by gracescharf**
# Description
* Creates very basic table to display key summary stats dummy data with formatted numbers
* Also installs Sass 
* Not a ton of styling done

### Note
I hate `<table>`s and usually avoid them but specifically chose one so that the HTML is semantic + the data is represented in a way that is good for accessibility. Also I'm guessing `<table>`s are going to be a big part of my FINTECH career

# Associated Issue
[Feature/Summary Stats](#1 )

# Testing
1. Switch into branch
2. Run npm i
3. Switch to localhost:3000
4. 👏 

# Screenshot 

image

### CODE REVIEW

File: components/SummaryStats.tsx [hunk 1]
```diff
@@ -1,35 +1,48 @@
...
-        <h3>Documentation &rarr;</h3>
-        <p>Find in-depth information about Next.js features and API.</p>
-      </a>
-      <a href="https://nextjs.org/learn" className={styles.card}>
-        <h3>Learn &rarr;</h3>
-        <p>Learn about Next.js in an interactive course with quizzes!</p>
-      </a>
+export default function SummaryStats() {
+  const toUSD = (num: number):string => {
+    return new Intl.NumberFormat('en-US', {style: 'currency', currency: 'USD'}).format(num)
```
Comment (iprado78) [inline components/SummaryStats.tsx]: // library
numUtils.ts

const { format: toUSD}  = new ...

export { toUSD }

File: components/SummaryStats.tsx [hunk 2]
```diff
@@ -1,35 +1,48 @@
 import React from 'react'
-import styles from '../styles/Home.module.css'
+import keyStats from '../data/keyStats.json' // todo: replace dummy data with api data
+import styles from '../styles/Summary.module.scss'
```
Comment (iprado78) [inline components/SummaryStats.tsx]: // SummaryStats -> directory
index.ts
SummaryStats.tsx
SummaryStats.module.scss

File: components/SummaryStats.tsx [hunk 3]
```diff
@@ -1,35 +1,48 @@
 import React from 'react'
-import styles from '../styles/Home.module.css'
+import keyStats from '../data/keyStats.json' // todo: replace dummy data with api data
```
Comment (iprado78) [inline components/SummaryStats.tsx]: Move data into SummaryStats folder
SummaryStats.mockData.ts

File: components/SummaryStats.tsx [hunk 4]
```diff
@@ -1,35 +1,48 @@
...
-        href="https://vercel.com/new?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-        className={styles.card}
-      >
-        <h3>Deploy &rarr;</h3>
-        <p>
-          Instantly deploy your Next.js site to a public URL with Vercel.
-        </p>
-      </a>
+  return (
+    <div>
```
Comment (iprado78) [inline components/SummaryStats.tsx]: Define a SummaryStats interface

interface SummaryStats {
...
}

File: components/SummaryStats.tsx [hunk 5]
```diff
@@ -1,35 +1,48 @@
...
+      <h2>{keyStats.companyName}</h2>
+     <table className={styles.summary}>
+       <caption>Summary</caption>
+       <tbody>
+        <tr>
```
Comment (iprado78) [inline components/SummaryStats.tsx]: Define a configuration object

Keys would be the keys of the interface
headerName -> "your text here"

Object.entries().map( ([key, value] ) => <tr>..</tr>)

**Review by dwalker11 — APPROVED** [review]

File: pages/index.js [hunk 6]
```diff
@@ -4,7 +4,7 @@ import Header  from "../components/Header"
 import AppContainer from "../components/AppContainer"
 import Main  from "../components/Main"
 import  CompanyInfo  from "../components/CompanyInfo"
-import SummaryStats from "../components/SummaryStats"
+import SummaryStats from "../components/SummaryStats/"
```
Comment (gracescharf) [inline pages/index.js]:
```suggestion
import SummaryStats from "../components/SummaryStats"
```

**Pull request merged**